lib/matplotlib/testing/conftest.py

```diff
@@ -82,7 +82,20 @@ def mpl_test_settings(request):
 
 @pytest.fixture
 def pd():
-    """Fixture to import and configure pandas."""
+    """
+    Fixture to import and configure pandas.
```

Comment:
This should mention that it makes pandas optional.

Reply:
```suggestion
    Fixture to import and configure the optional dependency pandas.
    
    The fixture is basically equal to `pd = pytest.importorskip('pandas')`
    plus some additional configuration.
    
    Use this whenever you need the `pandas` namespace in tests.
```